### PR TITLE
Update UG with advice from CS2101 consultation

### LINKED ISSUE #243: Add to UG that "clear" is an undoable command

### LINKED ISSUE #244: Add to UG ClearCommand#execute
Clear Command:
If you want to clear projects as well,
Add a line for ClearCommand#execute

### LINKED ISSUE #250: Update UG with advice from CS2101 consultation

### PR DESCRIPTION
**Pull request by vevek**
Closes #250 , closes #243, closes #244

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -17,15 +17,17 @@ https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/179/commits/aec461182c194c9ca2
...
+- Today View that surfaces an overview of upcoming events and deadlines 
 
-With CoLAB, you can efficiently manage all your school projects through our comprehensive project management tools. We have put in a lot of time and effort into designing a user-friendly User Interface (UI) such that CoLAB remains intuitive to both first-time users and seasoned ones. We look forward to seeing what you accomplish with CoLAB with a clean and inviting UI. What are you waiting for? Get your journey started with the [Quick Start section](#3quick-start)!
+It is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, CoLAB can get your project management tasks done faster than traditional GUI apps.
+
+With CoLAB, you can efficiently manage all your school projects through our comprehensive project management tools. We have designed a user-friendly User Interface (UI) such that CoLAB remains intuitive to both first-time users and seasoned ones. We look forward to seeing what you accomplish with CoLAB with a clean and inviting UI. Get your journey started with the [Quick Start section](#3quick-start)!
```
Comment (samuelfangjw) [inline docs/UserGuide.md:30]: `We look forward to seeing what you accomplish with CoLAB with a clean and inviting UI.`  Think  this part doesn't flow very well, can't see  the link between the two statements. Can we shorten this to `We look forward to seeing what you accomplish with CoLAB` and throw the second part somewhere else where we talk about the ui

File: docs/UserGuide.md [hunk 2]
```diff
@@ -598,11 +636,17 @@ Examples:
 
 ### **4.3 Navigating the UI**
 
-Although most of the buttons you see on the screen are clickable, the UI has been designed primarily to be navigated using the command line interface.
+Although most of the buttons you see on the screen are clickable, the UI has been designed primarily to be navigated using the command line interface. The commands listed in this section will help you navigate the UI without leaving keyboard.
+
+#### Viewing Today Panel : `today`
```
Comment (samuelfangjw) [inline docs/UserGuide.md:641]: Thinking of moving these  commands into their own individual section and create a section for today. While today section will only have one command, we can use that section to explain what today means  and what information is shown there.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -622,31 +666,23 @@ Examples:
 
 #### Viewing the overview of a project: `overview`
```
Comment (samuelfangjw) [inline docs/UserGuide.md:667]: Thinking of moving these 2 sections into projects

**Review by samuelfangjw — COMMENTED** [review]
Some comments on what I think we should change, open to hear your thoughts on this.

File: docs/UserGuide.md (hunk 1 above)
Comment (vevek) [reply docs/UserGuide.md:30]: It is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, CoLAB can get your project management tasks done faster than traditional GUI apps.

We can add `with a clean and inviting UI.` to this paragraph or remove it all together.

File: docs/UserGuide.md (hunk 2 above)
Comment (vevek) [reply docs/UserGuide.md:641]: Good idea. I'll shift it around.

File: docs/UserGuide.md (hunk 3 above)
Comment (vevek) [reply docs/UserGuide.md:667]: Good idea. I'll shift it around.

**Review by samuelfangjw — APPROVED** [review]
